platforms/hyperledger-besu/configuration/roles/create/genesis/ibft/tasks/main.yaml

```diff
@@ -32,16 +32,6 @@
   loop_control:
     loop_var: org
 
-# This task creates the genesis.json file from the template
-- name: Create genesis.json
-  template:
-    src: "ibftGenesisFile.tpl"
```

Comment:
Why is removal of genesis creation a solution? How will the genesis be created then?

Reply:
The genesis will be generated in this task
https://github.com/hyperledger/bevel/blob/main/platforms/hyperledger-besu/configuration/roles/create/crypto/ibft/tasks/main.yaml#L120
When creating it again and modifying it in the deleted taskk is when the problems occur when performing the tests with postman. This way it works the same as in the case of QBFT.